src/Util/DigitizeUtil/DigitizeUtil.js

```diff
@@ -0,0 +1,81 @@
+import OlObservable from 'ol/observable';
+
+/**
+ * This class provides some static methods which might be helpful when working
+ * with redlining functions.
+ *
+ * @class DigitizeUtil
+ */
+class DigitizeUtil {
```

Comment:
Would a more appropriate name be `FeatureUtil`?

Reply:
Renamed to `AnimateUtil` in 0504b8e. I think it's even more meaningful as `FeatureUtil`